apiary.apib

```diff
@@ -445,6 +445,48 @@ Retrieve all Business Asset Attribute Types for a specified Business Asset Type
 
 + Response 204
 
+# Group Manufacturers
+
+## Manufacturer Collection [/accounts/{accountId}/manufacturers{?page,pagesize,orderby,purpose,includeAudit}]
++ Parameters
```

Comment:
Updated for the items in the scope of this story; however, the majority of the collections are missing this documentation.